molecule/cloud-aws-sapmachine-13/molecule.yml

```diff
@@ -0,0 +1,103 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: requirements.yml
+driver:
+  name: ec2
+lint:
+  name: yamllint
+  options:
+    config-file: .yamllint
+platforms:
+  - name: test-aws-centos7-sapmachine-jdk-tarball-13
+    image: ami-9887c6e7
+    platform: centos7
+    instance_type: m5.large
+    region: us-east-1
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
+    assign_public_ip: false
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: centos
+    groups:
+      - rhel_family
+      - sapmachine
+      - jdk
+      - sapmachine_fallback
+
+  - name: test-aws-Debian9-sapmachine-jre-tarball-13
+    image: ami-003f19e0e687de1cd
+    platform: debian9
+    instance_type: m5.large
+    region: us-east-1
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
+    assign_public_ip: false
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: admin
+    groups:
+      - debian_family
+      - sapmachine
+      - jre
+      - sapmachine_fallback
+
+  - name: test-aws-Debian9-sapmachine-s3-13
+    image: ami-003f19e0e687de1cd
+    profile_name: epmldi-instance-profile
+    platform: debian9
+    instance_type: m5.large
+    region: us-east-1
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
+    assign_public_ip: false
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: admin
+    groups:
+      - debian_family
+      - sapmachine
+      - s3
+      - pip2
```

Comment:
What is this group for?

Reply:
This group is unnecessary. I will fix it!